conformance-test/retryStrategy.ts

```diff
@@ -254,7 +235,7 @@ async function createFileForTest(
 }
 
 function generateName(storageMethodString: String, bucketOrFile: string) {
-  return `${TESTS_PREFIX}${storageMethodString.toLowerCase()}${bucketOrFile}`;
+  return `${TESTS_PREFIX}${storageMethodString.toLowerCase()}${bucketOrFile}.${shortUUID()}`;
```

Comment:
Adding an extra UUID on the end to enforce uniqueness. This eliminates the need to delete buckets at the end of each test and saves a few network calls.

Reply:
shouldnt text prefix + storagemethodstring be globally unique? which case is this covering?

Reply:
Currently test prefix is defined as a constant at the file level. As a result every. test executing in this file utilizes the same prefix and because some tests are repeated in different scenarios the current setup does not guarantee uniqueness.